Documentation/operations/upgrade.rst

```diff
@@ -191,57 +191,6 @@ version which was installed in this cluster. Valid options are:
         --namespace=kube-system \\
         --set upgradeCompatibility=1.X
 
-.. note::
-
-   Make sure that you are using the equivalent options as for the initial
-   deployment. Cilium 1.9 renamed many of the Helm options to better support
-   specifying Cilium configuration via a ``values.yaml`` file. Consult the
-   `1.9_helm_options` to determine the 1.9 equivalent options for options you
-   previously specified when initially installing Cilium.
-
-   For example, an 1.8 installation with the following options::
-
-      --namespace=kube-system \\
-      --set global.k8sServiceHost=API_SERVER_IP \\
-      --set global.k8sServicePort=API_SERVER_PORT \\
-      --set config.ipam=kubernetes \\
-      --set global.kubeProxyReplacement=strict
-
-
-   Can be upgraded using the options below::
-
-      --namespace=kube-system \\
-      --set upgradeCompatibility=1.8 \\
-      --set k8sServiceHost=API_SERVER_IP \\
-      --set k8sServicePort=API_SERVER_PORT \\
-      --set ipam.mode=kubernetes \\
-      --set kubeProxyReplacement=strict
-
-   Instead of using ``--set``, you can also save the values relative to your
-   deployment in a YAML file and use it to regenerate the YAML for the latest
-   Cilium version. Running any of the previous commands will overwrite
-   the existing cluster's `ConfigMap` so it is critical to preserve any existing
-   options, either by setting them at the command line or storing them in a
-   YAML file, similar to:
-
-   .. code-block:: yaml
-
-      agent: true
-      upgradeCompatibility: "1.8"
-      ipam:
-        mode: "kubernetes"
-      k8sServiceHost: "API_SERVER_IP"
-      k8sServicePort: "API_SERVER_PORT"
-      kubeProxyReplacement: "strict"
-
```

Comment:
I'd like to rework the docs at some point to encourage the use of values.yaml files like this rather than passing in all the args manually since it's easier to compose and easier to manage over time, but yeah the upgrade guide isn't necessarily the correct place to push on this. :+1:

Reply:
oh this was actually removed by mistake I thought it was part of the 1.9 helm options. I'll re-add it.